android/libraries/rib-coroutines/README.md

```diff
@@ -0,0 +1,37 @@
+# rib-coroutines
+
+This module is responsible for defining the coroutines extensions for the rib-base module.
+
+## Installation
+```gradle
+dependencies {
+  implementation 'com.uber.rib:rib-coroutines:0.13.0'
+}
+```
+
+## Usage
+
+RibCoroutinesConfig is a global configuration object that allows for setting custom Dispatchers and Exception Handlers. This is useful for cases like Testing or when you are unable to inject Dispatchers.
+For configuration in app, this should be configured in the Application onCreate() before usage.
+
+```kotlin
+RibCoroutinesConfig.dispatchers = DefaultRibDispatchers()
+
+RibCoroutinesConfig.exceptionHandler = CoroutineExceptionHandler { _, exception ->
+    Log.e(exception)
+}
+```
+
+
+Lifecycle based components such as Interactors and Workers receive a dedicated coroutineScope for usage.
```

Comment:
That's a good opportunity to clarify that the `coroutineScope` is cancelled when the `ScopeProvider` completes.